www/front_src/src/Resources/Listing/columns/State/DetailsTable/Downtime.tsx

```diff
@@ -55,7 +55,7 @@ const DowntimeDetailsTable = ({ endpoint }: Props): JSX.Element => {
       id: 'is_fixed',
       label: t(labelFixed),
       type: ColumnType.string,
-      getContent: ({ is_fixed }): string => (is_fixed ? labelYes : labelNo),
+      getContent: ({ is_fixed }): string => t(is_fixed ? labelYes : labelNo),
```

Comment:
you could use `getYesNoLabel` to be consistent with acknowledgement

Reply:
Well done ! I did it !    :)